unit-05-input-types/exercises.ipynb

```diff
@@ -0,0 +1,351 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "919f3cb0-dd11-41f4-88a3-217ddedf456d",
+   "metadata": {},
+   "source": [
+    "### Unit 05 - Input, Data Type Conversion, Type Check\n",
+    "### In-Class Examples & Exercises"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2ca1796a",
+   "metadata": {
+    "colab_type": "text",
+    "id": "view-in-github"
+   },
+   "source": [
+    "<!--\n",
+    "    Instructors - This is a Google Colab link for running the notebook in a browser.\n",
+    "    The link will not be valid until the PR is merged to main, but you can swap \"main\" in the URL with the PR's branch name to try it.\n",
+    "-->\n",
+    "If you want to play with this when you don't have your Pi515 laptop: <a href=\"https://colab.research.google.com/github/pi515/tech-mentorship-2425/blob/main/unit-05-input-types/exercises.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Google Colab\"/></a>"
```

Comment:
I added a Google Colab link so people can play with these exercises even if they don't have their Pi515 laptop.

As mentioned in the code comment the URL isn't valid until this is merged to main. Here is a link with the URL referencing the branch from this PR (unit-05-input-types) - https://colab.research.google.com/github/pi515/tech-mentorship-2425/blob/unit-05-input-types/unit-05-input-types/exercises.ipynb